python/tests/unit/test_cache.py

```diff
@@ -465,6 +465,54 @@ def test_long_cluster_1():
 
 MadnessVerifier = Callable[[Tuple[IR, EnvoyConfig]], bool]
 
+def test_mappings_same_name_delta():
+    # Tests that multiple Mappings with the same name (but in different namespaces)
+    # are properly added/removed from the cache when they are updated.
+    builder = Builder(logger, "cache_test_4.yaml")
+    b = builder.build()
+    econf = b[1]
+    econf = econf.as_dict()
+
+    # loop through all the clusters in the resulting envoy config and pick out two Mappings from our test set (first and lase)
```

Comment:
Do we have guarantee's that our clusters are generated in consistent order? Clusters is a list so I would assume it would be just thought I would ask the question..

Reply:
I'm "pretty sure" that the order of clusters is deterministic. That said, for test it doesn't really matter if they are or not. Its fine if there are other clusters, we just assert that they each have a name since that is a blatant failure regardless.